install/semaphore-install.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 community-scripts ORG
+# Author: kristocopani
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+source /dev/stdin <<<"$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y \
+  curl \
+  mc \
+  git \
```

Comment:
Why do you need git here?

Reply:
This is because of this
![image](https://github.com/user-attachments/assets/51f070d4-a16f-4afd-b94b-3625c5b04a70)

Reply:
Wtf ^^ okay

Reply:
hehe 😁